### DISCUSSION TITLE
Is ParallelizableTest stable?

### DISCUSSION THREAD

**Discussion by yasminwz** [junit-team/junit-framework#3111] [General]
Hello, good afternoon, how are you? I am currently working on a project and I noticed the need to use ParallelizableTest in this application to improve performance, I would like to know if this feature is still experimental or is already more stable for use, thanks in advance to the JUNit team.

**Comment by sormuras** [comment]
[Parallel Execution](https://junit.org/junit5/docs/current/user-guide/#writing-tests-parallel-execution) is still an experimental feature of JUnit Jupiter. Yet, it is stable enough to be used ... for testing (-:

There's no `ParallelizableTest` class in [JUnit Jupiter's API](https://junit.org/junit5/docs/current/api/).
![image](https://user-images.githubusercontent.com/2319838/208981034-8cfe4adb-a2b4-4900-82f3-fe76745225e5.png)

Did you find it here? https://mikemybytes.com/2021/11/24/pragmatic-test-parallelization-with-junit5/

**Reply by yasminwz** [reply]
This annotation was created by me (sorry I didn't know the name wasn't this LOL), but in the documentation it says Parallel Execution 2.20
